src/pages/home/sidebar/OptionRow.js

```diff
@@ -129,6 +129,8 @@ const OptionRow = (props) => {
         },
     );
 
+    const avatarTooltips = props.showTitleTooltip ? _.pluck(displayNamesWithTooltips, 'tooltip') : undefined;
+
```

Comment:
```suggestion
    const avatarTooltips = props.showTitleTooltip && !props.option.isChatRoom && !props.option.isArchivedRoom ? _.pluck(displayNamesWithTooltips, 'tooltip') : undefined;

```

Reply:
Thanks, for this. I totally missed `props.option` object.